ext/mini_racer_extension/mini_racer_extension.cc

```diff
@@ -1185,6 +1234,31 @@ VALUE rescue_callback(VALUE rdata, VALUE exception) {
     return exception;
 }
 
+static void throw_ruby_error(Isolate* isolate, ContextInfo* context_info, VALUE rb_error) {
+        VALUE message = rb_funcall(rb_error, rb_intern("message"), 0);
+	HandleScope scope { isolate };
+
+	Local<ObjectTemplate> errorTemplate = context_info->isolate_info->ruby_error_template.Get(isolate);
+	Local<Object> errorInstance;
+	Local<Context> context = context_info->context->Get(isolate);
+	if(!errorTemplate->NewInstance(context).ToLocal(&errorInstance))
+		return;
+	/* TODO: Is this cast sane? */
+	errorInstance->SetInternalField(0, External::New(isolate, (void*) rb_error));
```

Comment:
Yeah, I have no idea if you can safely shove the ruby error VALUE into the field here. Seems unlikely, especially since the function is looking for a v8 Local<Value>.

Reply:
https://github.com/ruby/ruby/blob/cb4e2cb55a59833fc4f1f6db2f3082d1ffcafc80/include/ruby/internal/value.h#L40
https://www.reddit.com/r/C_Programming/comments/dztsbu/cast_from_void_to_uintptr_t/

not 100% safe, but mostly safe. :)
Which is good enough for me!

Reply:
Wait-
Now that I've read and understood the code more, we don't use this data at all, right? We've already serialized this into the JS-side error RubyError and have thrown it. Are we using it somewhere I missed?

Reply:
The value stored here is currently unused. This is mainly prep for moving away from storing the ruby exception in `@current_exception` on the context.